cmd/bootstrap/cmd/machine_account_key_test.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/bootstrap"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	ioutils "github.com/onflow/flow-go/utils/io"
+)
+
+func TestMachineAccountKeyFileExists(t *testing.T) {
+	var keyFileExistsRegex = `^machine account private key already exists`
+
+	dirName := strconv.FormatInt(time.Now().UnixNano(), 10)
+	regex := regexp.MustCompile(keyFileExistsRegex)
+
+	// command flags
+	flagOutdir = fmt.Sprintf("/tmp/%s", dirName)
```

Comment:
We have a utility which creates then tears down a temporary directory we can use here.
https://github.com/onflow/flow-go/blob/ff92e3b5a8edfb80eb797e8896c8c2a0bdea8fcd/utils/unittest/unittest.go#L172